cmd/keymasterd/config.go

```diff
@@ -40,6 +41,8 @@ type baseConfig struct {
 	TLSCertFilename              string   `yaml:"tls_cert_filename"`
 	TLSKeyFilename               string   `yaml:"tls_key_filename"`
 	SSHCAFilename                string   `yaml:"ssh_ca_filename"`
+	UnsealPasswordSecretArn      string   `yaml:"unseal_password_secret_arn"`
```

Comment:
For clarity these should be in a separate section. (will make the names more readable). I would say:
type SelfUnsealing struct {
        AWSSecretName string `yaml:"aws_secret_name"`
        AWSSecertKey ....
}

And also I would use a name instead of an arn so that I could use the same config across aws regions.

Reply:
Done.